gateway/src/commands/init.rs

```diff
@@ -79,9 +82,13 @@ pub struct Init {
     statistics_service_url: Option<url::Url>,
 
     /// Allows this gateway to run an embedded network requester for minimal network overhead
-    #[clap(long)]
+    #[clap(long, conflicts_with = "with_ip_forwarder")]
```

Comment:
should they be conflicting with each other? in principle, since they have different addresses, why can't we allow both?

Reply:
In principle yes, in practice, for now I'm taking shortcuts elsewhere to assume this is strictly either